colin-api/src/colin_api/models/business.py

```diff
@@ -153,7 +153,7 @@ def get_colin_identifier(cls, lear_identifier, legal_type):
             cls.TypeCodes.BC_COMP.value,
             cls.TypeCodes.ULC_COMP.value,
             cls.TypeCodes.CCC_COMP.value
-        ]:
+        ] and lear_identifier.startswith('BC'):
             return lear_identifier[-7:]
```

Comment:
Do  you need to convert the Cxxxxxxx identifiers?

Reply:
nope. only for BC prefix